framework/helpers.php

```diff
@@ -54,18 +66,19 @@ function cpt_next_posts_link( WP_Query $wp_query, $label ) {
 	}
 
 	if ( $nextpage <= $max_page ) {
-		$attr = apply_filters( 'next_posts_link_attributes', '' );
-		$attr = apply_filters( 'jtf_next_posts_link_attributes', $attr );
+		$attr           = apply_filters( 'next_posts_link_attributes', '' );
+		$attr           = apply_filters( 'jtf_next_posts_link_attributes', $attr );
+		$load_more_attr = apply_filters( 'jtf_load_more_attributes', $load_more_attr );
```

Comment:
if you use this as separate attr list - you can set class 'jtf_load_more' for the link. or it will be not valid, because next posts link attributes generate the class attribute